pkgs/development/libraries/itk/generic.nix

```diff
@@ -21,6 +21,13 @@ stdenv.mkDerivation rec {
     hash = "sha256-deJbza36c0Ohf9oKpO2T4po37pkyI+2wCSeGL4r17Go=";
   };
 
+  itkSimpleITKFiltersSrc = fetchFromGitHub {
+    owner = "InsightSoftwareConsortium";
+    repo = "ITKSimpleITKFilters";
+    rev = "bb896868fc6480835495d0da4356d5db009592a6";
```

Comment:
How is that commit determined? Seems to be a general thing. Also we skills nice this variable into a let in to not accidentally turn it into a bash variable

Reply:
I've lifted these sources into a `let`.

The commits come from the cmake files (but they seemingly aren't too sensitive e.g. both versions of ITK are currently sharing the same versions of these remote modules).  I opened an issue for this [here](https://github.com/NixOS/nixpkgs/issues/222929).